subclass/Aevilok; Craftsman.json

```diff
@@ -0,0 +1,369 @@
+{
+	"_meta": {
+		"sources": [
+			{
+				"json": "roguecraft",
+				"abbreviation": "A:RC",
+				"full": "Craftsman",
+				"authors": [
+					"Aevilok"
+				],
+				"convertedBy": [
+					"hakr14"
+				],
+				"version": "1.0",
+				"url": "https://www.gmbinder.com/share/-Lm48Z9HxvhHMgblnjz7"
+			}
+		],
+		"dateAdded": 1643685011,
+		"dateLastModified": 1643764459
+	},
+	"subclass": [
+		{
+			"name": "Craftsman",
+			"source": "roguecraft",
+			"shortName" : "Craftsman",
+			"subclassFeatures": [
+				"Craftsman|Rogue||Craftsman|roguecraft|3",
+				"Fast Hands|Rogue||Craftsman|roguecraft|9",
+				"Windup Familiar|Rogue||Craftsman|roguecraft|13",
+				"Preparation Work|Rogue||Craftsman|roguecraft|17"
+			],
+			"className": "Rogue",
+			"classSource": "PHB"
+		}
+	],
+	"subclassFeature": [
+		{
+			"name": "Craftsman",
+			"source": "roguecraft",
+			"page": 1,
+			"className": "Rogue",
+			"classSource": "PHB",
+			"subclassShortName": "Craftsman",
+			"subclassSource": "roguecraft",
+			"level": 3,
+			"entries": [
+				"While some rogues prefer to focus on their agility or accuracy, the Craftsmen are the rogues that like to solve their problems using any tool they can get at their disposal. These rogues face problems with clever thinking, and an even more clever use of the items they can find, making makeshift ropes, tents, or bags of caltrops to execute their plan.",
+				{
+					"type": "refSubclassFeature",
+					"subclassFeature": "Mundane Creations|Rogue||Craftsman|roguecraft|3"
+				},
+				{
+					"type": "refSubclassFeature",
+					"subclassFeature": "Junkyard Survival|Rogue||Craftsman|roguecraft|3"
+				}
+			]
+		},
+		{
+			"type": "entries",

```

Comment:
`"type": "entries",` isn't allowed on the root of a `subclassFeature`, as it is implicit. Delete this line

Reply:
Well that's... odd. I have another homebrew that uses `"type": "entries"` for the subfeatures, and that works... The file worked fine when I uploaded it to homebrew manager manually too...
Well, in either case, the issue here is fixed. Thanks!